external/footer.php

```diff
@@ -7,15 +7,15 @@
     $isAjax = true;
 }
 
-if (extension_loaded('xhprof') && $_xhprof['doprofile'] === true) {
+if ( $_xhprof['ext_name'] && $_xhprof['doprofile'] === true) {
     $profiler_namespace = $_xhprof['namespace'];  // namespace for your application
-    $xhprof_data = xhprof_disable();
+    $xhprof_data = call_user_func($_xhprof['ext_name']."_disable");
```

Comment:
Any reason for double quote usage (I see you mix match usage of single and double quotes for no reason). Better to use single quotes all the time.